src/Commands/Self/Plugin/InstallCommand.php

```diff
@@ -42,6 +43,32 @@ public function install(array $projects)
         }
     }
 
+    /**
+     * Import Terminus 2 plugins.
+     *
+     * @command self:plugin:import
```

Comment:
In the "naming things is hard" department: plugin:import, plugin:migrate, something else? I think maybe I like migrate.

Reply:
+1 for migrate

Reply:
I like migrate too

Reply:
By popular demand and poll results, migrate is the winner.